### PR TITLE
Remove sender id slot

### PR DESCRIPTION
**Pull request by bscheltinga**
Fixes https://github.com/PerfectFit-project/virtual-coach-issues/issues/155

### CODE REVIEW

File: Rasa_Bot/tests/test_conversations.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: "2.0"
+version: "3.0"
```
Comment (svenvanderburg) [inline Rasa_Bot/tests/test_conversations.yml:1]: Did Nele forget this in updating to 3.0? Nice catch :)

**Review by svenvanderburg — APPROVED** [review]
Nice! This simplifies things a lot!

To double check: is the model you included a rasa 3.0 model?

File: Rasa_Bot/tests/test_conversations.yml (hunk 1 above)
Comment (bscheltinga) [reply Rasa_Bot/tests/test_conversations.yml:1]: Yes, it gave an error with the "Rasa CI / Train and Test model" test. But I tried to revert the change as I thought it should be another issue. But apparently without succes...

Comment (bscheltinga) [reply Rasa_Bot/tests/test_conversations.yml:1]: And I should check whether I did use rasa v3 and maybe retrain the models.

Comment (svenvanderburg) [reply Rasa_Bot/tests/test_conversations.yml:1]: No worries, just leave the change in this PR.

Comment (bscheltinga) [reply Rasa_Bot/tests/test_conversations.yml:1]: The thing was actually that I did not use rasa 3. I retrained the model with v3 and tested it again

**Pull request merged**